manim/scene/scene.py

```diff
@@ -223,15 +222,6 @@ def add(self, *mobjects):
         self.mobjects += mobjects
         return self
 
-    def add_mobjects_among(self, values):
```

Comment:
I did not know this method existed.
Probably something like:
```python
class ABC(Scene):
    def construct(self):
        dot = Dot().set_color(GREEN)
        cric= Circle()
        sq= Square()
        self.add_mobjects_among(locals().values())
```
could be useful in some situations, but I am also fine with removing these.

Reply:
This is super unpythonic, and I'm very much against maintaining something like this.

Reply:
okay!